core/src/test/java/org/apache/calcite/test/CalciteAssert.java

```diff
@@ -672,7 +672,7 @@ static void assertPrepare(
   }
 
   /** Converts a {@link ResultSet} to a string. */
-  static String toString(ResultSet resultSet) throws SQLException {
+  public static String toString(ResultSet resultSet) throws SQLException {
```

Comment:
Not sure this method was intended to be public, but right now, can't think of a better alternative.